src/operator/contrib/ctc_loss-inl.h

```diff
@@ -50,11 +50,10 @@
 
 namespace mxnet {
 namespace op {
-
 namespace ctc_loss {
-enum CTCLossOpInputs { kData, kLabel };
-enum CTCLossOpOutputs { kOut, kGrad };
-enum CTCLossOpForwardResource { kTempSpace };
+  enum CTCLossOpInputs { kData, kLabel };
+  enum CTCLossOpOutputs { kOut, kGrad };
+  enum CTCLossOpForwardResource { kTempSpace };
```

Comment:
which case does this part fall into? how were we able to check it in without breaking master build if it's either of the cases?

Reply:
Sorry, overlooked these lines. I have removed them.